packages/fast-usdc/src/exos/settler.js

```diff
@@ -206,19 +215,19 @@ export const prepareSettler = (
         ) {
           const { mintedEarly } = this.state;
           const { value: fullValue } = fullAmount;
+          // XXX i think this only contains Advancing txs - should this be a separate store?
           const key = makeMintedEarlyKey(forwardingAddress, fullValue);
           if (mintedEarly.has(key)) {
             mintedEarly.delete(key);
             if (success) {
-              void this.facets.self.disburse(
-                txHash,
-                forwardingAddress,
-                fullValue,
-              );
+              // TODO: does not write `ADVANCED` to vstorage
+              // this is the "slow" experience, but we've already earmarked fees so disburse
+              void this.facets.self.disburse(txHash, fullValue);
             } else {
+              // TODO: does not write `ADVANCE_FAILED` to vstorage
```

Comment:
```suggestion
              // FIXME: does not write `ADVANCE_FAILED` to vstorage
```

Reply:
TODO will appear in commit history, but not the final HEAD of this PR